aws/logs_monitoring/lambda_function.py

```diff
@@ -390,14 +399,24 @@ def datadog_forwarder(event, context):
 
     if DD_FORWARD_LOG:
         forward_logs(filter_logs(map(json.dumps, logs)))
+        DD_FORWARDER_TELEMETRY["logs_forwarded"]+=len(list(filter_logs(map(json.dumps, logs))))
 
     forward_metrics(metrics)
+    DD_FORWARDER_TELEMETRY["metrics_forwarded"]+=len(metrics)
```

Comment:
I think it makes more sense to have this logic happen inside the `forward_metrics` function, since it's a small detail which shouldn't /pollute the main (i.e., `datadog_forwarder`) function. I.e., when you read the main function, you can quickly see what this function does on the high level without being distracted by lots of "details".

Reply:
Refactored/cleaned up so the main function is clean